spec/fixtures/module/preload-sandbox.js

```diff
@@ -9,6 +9,15 @@
     window.process = process
     if (process.env.sandboxmain) {
       window.test = {
+        crash: typeof process.crash,
+        hang: typeof process.hang,
+        getCreationTime: typeof process.getCreationTime,
```

Comment:
getting the types of these doesn't validate that they actually work, which is IMO important for testing sandbox functions, since we've previously observed crashes that only occur when you actually call a function from the sandbox (b/c that's when the sandbox policy gets checked).

For `crash` and `hang` I'm willing to let it slide, but these other functions are easily testable.

Reply:
done

Reply:
@nornagon should I also check all the properties in the objects (as we do in `spec/api-process-spec.js`)?

Reply:
I'm fine with just checking that the function returned an appropriate object. as long as we call it!